tests/backup/backup_restore_basic_test.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/portworx/torpedo/pkg/log"
 	. "github.com/portworx/torpedo/tests"
 	v1 "k8s.io/api/core/v1"
+
+	semver "github.com/blang/semver"
```

Comment:
This is a redundant alias naming. Not required.

Reply:
✅